packages/flutter/lib/src/material/scrollbar_theme.dart

```diff
@@ -293,3 +325,177 @@ class ScrollbarTheme extends InheritedWidget {
   @override
   bool updateShouldNotify(ScrollbarTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The leading and trailing Scrollbar buttons that are simple arrows.
+class ArrowScrollbarButtonsShape extends ScrollbarButtonsShape {
+  /// This abstract const constructor enables subclasses to provide
+  /// const constructors so that they can be used in const expressions.
```

Comment:
Nit: this comment isn't necessary. It is not specific to this constructor and won't provide much info for someone reading the API docs.

Reply:
The CI request a document here :)